.github/ISSUE_TEMPLATE.yml

```diff
@@ -0,0 +1,55 @@
+name: Bug Report
+literal_block: |
+  > If you have encountered a security-related issue with Yii, please use the [security issue form](https://www.yiiframework.com/security) to report any security issues you have encountered to us.
+  > ***WARNING!*** DO NOT use the issue tracker or discuss it in the public forum as it will cause more damage than help.
+description: Create a bug report
+labels: ["defect"]
+body:
+  - type: input
+    id: yiiversion
+    attributes:
+      label: Yii Version
```

Comment:
This is package template. This field not need.

Reply:
But add/maintain this in other packages?

Reply:
If we use separate fields, then this should be conrecte package version. For example, "Yii DB version".